modules/library/src/main/scala/zio/elasticsearch/HttpElasticExecutor.scala

```diff
@@ -255,7 +295,7 @@ private[elasticsearch] final class HttpElasticExecutor private (config: ElasticC
                 getQueryParams(List((Scroll, Some(ScrollDefaultDuration)), ("routing", s.routing)))
               )
             )
-            .response(asJson[ElasticQueryResponse])
+            .response(asJson[SearchWithAggsResponse])
```

Comment:
In this case we're dealing with a search request with no aggregations. Is there a need to parse the response to a type that includes aggregations or could we easily separate the response types?